packages/zowe-explorer/i18n/sample/src/Profiles.i18n.json

```diff
@@ -1,5 +1,6 @@
 {
   "profiles.operation.cancelled": "Operation Cancelled",
+  "profiles.manualEditMsg": "Editing or removal of profiles will need to be done manually.",
```

Comment:
Thanks for adding this message 👍  Should we prefix it with something like `Config file has been opened in the editor.`? I know that should be obvious, but we want to make sure that users don't miss this and think they have to manually track down the location of the file and open it.